tools/test.py

```diff
@@ -83,7 +84,10 @@
                           
         parser.add_argument("--test-spec", dest="test_spec",
                           default=None, help="Destination path for a test spec file that can be used by the Greentea automated test tool")
-        
+        parser.add_argument("--export", dest="export",  action="store_true",
+                          default=False, help="Export a project")
+        parser.add_argument("-i", dest="ide", 
+                          default=None, help="IDE")
```

Comment:
Can this line be made to be closer to what is used in `project.py`? See here: https://github.com/mbedmicro/mbed/blob/master/tools/project.py#L37

That will do the IDE argument validation.